### PR TITLE
Sc 33536 - Add QHack Banner

### PR DESCRIPTION
**Pull request by Anakhom**
https://app.shortcut.com/xanaduai/story/33536/add-banner-to-advertise-qhack-on-the-pennylane-ai

- Added QHack banner to Landing page:

https://user-images.githubusercontent.com/58717051/217952777-f3c50077-ea02-4582-8217-ff54cc02b28d.mov

### CODE REVIEW

**Review by doctorperceptron — APPROVED** [review]
# :partying_face: 

LGTM!

File: jinja/index.html [hunk 1]
```diff
@@ -6,9 +6,35 @@
 <div class="parallax" style="background-image: url('img/background.png'); box-shadow: none;">
     <div class="container">
 
-        <div class="pt-5 text-center teal-text text-lighten-2">
+
+        <div class="pt-5 teal-text d-flex justify-content-center flex-wrap text-lighten-2">
+            <div class="banner-card bg-white mt-4 pt-3 pb-3 pr-4 pl-4">
+              <h3 class="text-center mt-0">QHack is here!</h3>
+              <div class="d-flex mt-2 mb-2 text-start justify-content-center align-items-center">
+                <img width="100px" src="img/QHack2023_logo_black.png" class="QHack_logo mr-4"/>
```
Comment (LucasSilbernagel) [inline jinja/index.html:14]:
```suggestion
                <img width="100px" src="img/QHack2023_logo_black.png" alt="QHack Logo" class="QHack_logo mr-4"/>
```

**Review by LucasSilbernagel — CHANGES_REQUESTED** [review]
Thanks @Anakhom ! Just one small comment.

**Review by LucasSilbernagel — APPROVED** [review]
🎉

**Review by josh146 — APPROVED** [review]
Thanks @Anakhom!

**Pull request merged**